ports/cxd56/supervisor/port.c

```diff
@@ -45,7 +47,8 @@
 safe_mode_t port_init(void) {
     boardctl(BOARDIOC_INIT, 0);
 
-    board_init();
```

Comment:
Either the call to board_init() should remain, or board_init() should be removed everywhere in this port.

Reply:
`board_init` is called here: https://github.com/adafruit/circuitpython/blob/47efd595fce11240f92efecf768800e8d81fa702/main.c#L437
Why should we call this function twice?

Reply:
It's my mistake, thanks.  This was moved in 100409961aa32e84810d6c86309e66277b4ea902 but the fix was not applied to cxd56 port.